content/en/docs/concepts/services-networking/_index.md

```diff
@@ -7,44 +7,214 @@ description: >
+
+{{< figure src="/docs/images/k8net-PodSameHost03.drawio.svg" alt="k8s pods samehost" class="diagram-large" caption="Figure 3. Example of Pod 1 and Pod 2 on Node 1 communicating with each other" >}}
+
+
+ 
+### Pods on different nodes
+
+Your Kubernetes deployment might place pods on a network of nodes. With this pod topology, you can distribute your workloads across many pods to increase scale and resiliency. Inter-pod networking between distinct nodes becomes a crucial function of your cluster network. 
+
+
+#### Container network interface
+
+Network plugins support container networking across pods and nodes in your cluster environment. You can think of network plugins as the software responsible for container runtime interface attachment and detachment, pod IP address allocation and network function configuration such as tunnel setup, BGP and IP routing.  
+
+{{< note >}}
+Network plugins are generally referred to as CNI plugins.
+{{< /note >}}
+
+CNI plugins adhere to the specifications defined in the [container network interface](https://www.cni.dev/). The rational behind the development of the CNI was to avoid code duplication, simplify container network configuration and operations, and guarantee cluster compatibility. Most importantly, it offers the Kubernetes community a uni specification for building and deploying container networking functions in cluster networks. 
+
+The CNI provides a [specification](https://www.cni.dev/docs/spec/), a Go [library for CNI application and integration](https://github.com/containernetworking/cni/blob/main/libcni) and a [command-line tool](https://github.com/containernetworking/cni/blob/main/cnitool) for executing CNI plugins. You can find reference plugins and a template in a [separate repository](https://github.com/containernetworking/plugins) if you wish to build your own. 
+
+You will not find a single CNI plugin that addresses all of your requirements. In practice, you will likely employ multiple plugins that in some cases can be "chained" together to support your environment. 
+
+To learn more about CNI and plugins see the following:
+
+* [Documentation](https://www.cni.dev/docs/)
+
+* [Specification](https://www.cni.dev/docs/spec/)
+
+* [Supported container runtimes](https://github.com/containernetworking/cni#container-runtimes)
+
+* [Reference plugins](https://www.cni.dev/plugins/current/#reference-plugins)
+
+* [3rd party plugins](https://github.com/containernetworking/cni#3rd-party-plugins)
+
+* [Introduction to CNI](https://youtu.be/YjjrQiJOyME)
+
+* [CNI deep dive](https://youtu.be/zChkx-AB5Xc)
+
+* [Webinar: Kubernetes and Networks](https://www.youtube.com/watch?v=GgCA2USI5iQ)
+
+The examples in the following section illustrate the use of CNI plugins in achieving container network connectivity.
+
+#### Pods on different nodes using a virtual overlay network
+
+This example uses a virtual overlay network to support inter-pod networking.  
```

Comment:
```suggestion
Because the two pods in this scenario are on two different nodes, packets first enter a local
virtual layer 2 bridge, then transit over an overlay network, and finally the layer
2 bridge on the target node. Your cluster might use a different approach.
```